Tests/TuistGenerateAcceptanceTests/GenerateAcceptanceTests.swift

```diff
@@ -201,6 +201,48 @@ final class GenerateAcceptanceTestiOSAppWithCustomResourceParserOptions: TuistAc
     }
 }
 
+final class GenerateAcceptanceTestIosAppWithInfoPlist: TuistAcceptanceTestCase {
+    func test_ios_app_with_multiline_string_value_infoplist() async throws {
+        try setUpFixture(.iosAppWithInfoplist)
+        try await run(GenerateCommand.self)
+        try await run(BuildCommand.self)
+        for resource in ["Test1.plist", "Test2.plist", "Test3.plist"] {
+            try await XCTAssertProductWithDestinationContainsResource(
+                "App.app",
+                destination: "Debug-iphonesimulator",
+                resource: resource
+            )
+        }
+
+        XCTAssertTrue(
+            try FileHandler.shared.readTextFile(
+                fixturePath.appending(components: "Derived", "Sources", "TuistPlists+App.swift")
+            )
+            .contains(
+                #"public static let items: [String] = ["A\\n"]"#
+            )
+        )
+
+        XCTAssertTrue(
+            try FileHandler.shared.readTextFile(
+                fixturePath.appending(components: "Derived", "Sources", "TuistPlists+App.swift")
+            )
+            .contains(
+                #"public static let items: [String] = ["BC"]"#
```

Comment:
Is this what you want? Should newlines be stripped away completely? Shouldn't this be `B\\nC`?

Reply:
@danyf90 
Yes. Since `replace` wasn't working, I changed the approach. 
I kept the `\n` or `\\n` that the user entered in the plist file, but removed the actual new line characters to prevent compilation failures. 
The main issue with this problem is that a build failure occurs due to improperly generated files.

Reply:
https://github.com/SwiftGen/StencilSwiftKit/blob/fb6a88bac60b088a9f8b61bbc70eacecb2100b59/Sources/StencilSwiftKit/Filters%2BStrings.swift#L259-L271

However, I am curious why the replace function did not work properly

Reply:
If a string with newline is not valid, failing compilation is better than automagically fixing it IMO